master/linking.html

```diff
@@ -679,7 +678,6 @@ <h4 id="processingURL-validity">Valid URL targets</h4>
   <!--
   <li>the <a href="color.html#ColorProfileSrcProperty">'src'</a> descriptor on an @color-profile definition must reference an ICC profile resource</li>
   -->
-  <li>the <a>'cursor property'</a> property must reference either a <a>'cursor element'</a> element or a document that can be processed as an image</li>
   <li>the <a>'fill'</a> property (see <a href="painting.html#SpecifyingPaint">Specifying paint</a> for reference rules)</li>
   <li>the <a>'marker property'</a>, <a>'marker-start'</a>, <a>'marker-mid'</a> and <a>'marker-end'</a> properties must reference a <a>'marker element'</a> element.</li>
   <li>the <a>'shape-inside'</a> and <a>'shape-subtract'</a> properties must reference an element type
```

Comment:
There's another reference to the cursor property accepting URL references to cursor elements in the paragraph after this list that needs to be removed.

Reply:
done